src/v3/plugins/github/urlIdParse.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+export function reviewUrlToId(url: string): string {
+  const result = new RegExp(
+    "^https://github.com/\\w+/\\w+/pull/\\d+#pullrequestreview-(\\d+)$"
```

Comment:
The `\w` character class does not include `-`, which is a valid
character in GitHub repositories (e.g., sourcecred/example-github).
Consider instead `[a-zA-Z0-9_-]` throughout this module, and consider
adding an `example-github` test.

Reply:
Great catch, thanks. I re-wrote the regex generation to be cleaner, and to handle this case. I used #98 as inspiration. Also, I changed the test case to include underscores, numbers, and dashes in the owner and repo name.